staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -140,6 +130,29 @@ func LogRequestObject(ae *audit.Event, obj runtime.Object, gv schema.GroupVersio
 			ae.ObjectRef.ResourceVersion = meta.GetResourceVersion()
 		}
 	}
+	// TODO: ObjectRef should include the API group.
+	if gvr.Version != "" {
+		ae.ObjectRef.APIVersion = gvr.Version
+	}
+	if gvr.Resource != "" {
+		ae.ObjectRef.Resource = gvr.Resource
```

Comment:
Also the logic is different than the one above: here we check gvr to be non-empty, above the event values. Should be the same if there is no good reason.